src/Elastic.Apm/Model/NoopTransaction.cs

```diff
@@ -39,6 +44,8 @@ public NoopTransaction(string name, string type, ICurrentExecutionSegmentsContai
 			ReusableContextInstance;
 
 		public Dictionary<string, string> Custom => _custom.Value;
+
+		public IConfigSnapshot ConfigSnapshot => _defaultConfigSnapshot;
```

Comment:
Wondering whether this should also be a real snapshot of configuration, since only `ITransaction` is publicly exposed; I could imagine a scenario where a user might decide to take some action based on a dynamic configuration value from central config, which wouldn't be seen with the stubbed `DefaultConfigSnapshot`

Reply:
Changed to use the real snapshot.